src/lib/ops/validation.js

```diff
@@ -47,7 +47,7 @@ const users = {
     organizationId: validationRules.organizationId
   },
   createUser: {
-    id: validationRules.id.optional().allow('').description('User ID'),
+    id: validationRules.id.optional().description('User ID'),
```

Comment:
I would keep it required and not use `uuid`.

I think we had a misunderstanding: my idea was to have the validation for a generic string in all ids (but team.id), not also the organization id to be optional.